cmd/gen-go-sample/main.go

```diff
@@ -338,63 +333,63 @@ func (g *generator) getInitInfo(inType pbinfo.ProtoType, methConf GAPICMethod, v
 	}
 
 	// Set up request object.
-	for _, def := range valSet.Parameters.Defaults {
-		if err := itree.parseInit(def, g.descInfo); err != nil {
-			return initInfo{}, errors.E(err, "can't set default value: %q", def)
+	for _, reqConf := range reqConfs {
```

Comment:
how about `_, fieldConf := range reqConf`? (because each `RequestConfig` describes a field)
I think that makes for clearer local var and param names, and avoids them looking so similar

Reply:
Changed to `fieldConf` and `fieldConfs`. (It's somewhat weird to use a singular noun `reqConf` as a name of a slice)

Reply:
Yeah, in general. But I'm thinking as a user of a "request config" being comprised of several "field configs", so in that context IMO it makes sense for the name to be singular. But not a biggie either way. :man_shrugging: